File: projects/store-or/src/main/java/com/robinfood/storeor/entities/PosEntity.java
package com.robinfood.storeor.entities;

import lombok.Builder;
import lombok.Data;

@Builder
@Data
public class PosEntity {

    private final String code;
    private final Long id;
    private final String name;
    private final PosTypeEntity posTypes;
    private final Boolean status;

}

File: projects/posv2/core/src/main/kotlin/com/robinfood/core/entities/userconfiguration/PosEntity.kt
package com.robinfood.core.entities.userconfiguration

data class PosEntity(

    val id: Long?,

    val name: String?,

    val currency: String?,

    val countryId: Long?,

    val isDelivery: Boolean?,

    val flowId: Long?,

    val isMultiBrand: Boolean?
)